tests/hooks/advocate-conversion-utils.spec.ts

```diff
@@ -15,22 +15,24 @@ type RecordFields = {
   city?: string,
   country?: string,
   region?: AdvocatesWorldRegion
-  // slackId: string
+  slackId: string,
+  slackUsername: string
```

Comment:
These properties are optional, right? Correct me if I'm wrong

Reply:
I think it depends on what we mean as optional.

These properties aren't required in the Airtable, but we also filter out any entries that don't have a `slackId`... so while not _required_ by any hard coded rule, I do think we require them since all advocate objects will have a `slackId` and `slackUsername`

By definition, does that make them required? or should this still be optional?

Reply:
For now, let's leave them as a "check" of only getting the advocates with `slackId`